src/boot/stylesheets/actions.scss

```diff
@@ -57,7 +57,7 @@
 	}
 }
 
-.wpnc__action-link {
+.wpnc__main .wpnc__action-link {
```

Comment:
We do, unfortunately, to override the `.wpnc__main button` in `main.scss`. I put the `button` declaration within `wpnc__main` in `main.scss` because every other element styling was done the same way (except for `body`) – _feels_ safer (and more consistent)?

I'll go ahead with this, and we can revisit later if we want to re-evaluate `main.scss`.